package.json

```diff
@@ -5,12 +5,14 @@
   "main": "index.js",
   "scripts": {
     "benchmark": "node benchmark",
```

Comment:
we'll have to change the directory that `benchmark/index.js` gets `tests.js` now that it moved